api/views/group.py

```diff
@@ -24,3 +29,19 @@ def get_serializer_class(self) -> type[BaseSerializer[Group]]:
             return PublicGroupWithStudentsSerializer
 
         raise NotImplementedError(f"Unknown action: {self.action}")
+
+    @action(detail=True, methods=["post"])
+    def start(self, request: Request) -> Response | HttpResponse:
+        data = request.data
```

Comment:
you can just call self.get_object() here, since it's detail=True action

Reply:
Done